lib/mail/core_extensions/string/access.rb

```diff
@@ -7,48 +7,6 @@
 # itself does not depend on ActiveSupport to avoid versioning conflicts
 
 class String
-  unless '1.9'.respond_to?(:force_encoding)
```

Comment:
These were unused as well. In #1094 I thought `iconv` returned a String but I confirmed that it returns an array so `first` is never called on String.

https://github.com/mikel/mail/blob/2b76d0a349895a84f02f1f86f762a4906bd133e8/lib/mail/encodings.rb#L153